### ISSUE TITLE
[Accessibility]The messages announced by screen reader can not be specified 

### ISSUE THREAD

**Issue by liuguangyicao** [MicrosoftDocs/msteams-docs#7861]
# Description
The messages announced by screen reader can not be specified when a new adaptive card is opened. In the video, when the adaptive card is opened, the screen read will announce "Ken.lin1104:test title" which is combination of the app name and the adaptive card title. But I only want to be announced with the title "test title". In a word, there is no field in the adaptive card support ing specifying the announce messages.

https://user-images.githubusercontent.com/25132507/211250604-5a75389b-ee4c-470e-a4d2-f066bc4d413e.mov

<img width="1323" alt="image4" src="https://user-images.githubusercontent.com/25132507/211250628-6432c4e3-a4b2-4269-b2b8-61cbed6edefe.png">

# Expected Behaviour
The announce messages when we firstly open the card can be specified by ourself.

# Actual Behaviour
The announce messages are supported by default and can not be specified.

# Card Json
```
{
  "type": "AdaptiveCard",
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.4",
  "body": [
    {
      "id": "topic1",
      "type": "Input.Text",
      "placeholder": "Placeholder",
      "label": "Label",
      "maxLength": 256,
      "isRequired": true,
      "errorMessage": "Test Error Message",
      "spacing": "Large"
    },
    {
      "type": "Input.ChoiceSet",

      "choices": [
        {
          "title": "Choice 1",
          "value": "Choice 1"
        },
        {
          "title": "Choice 2",
          "value": "Choice 2"
        }
      ],
      "id": "userIds",
      "isMultiSelect": true,
      "placeholder": "Placeholder",
      "label": "Label",
      "isRequired": true,
      "errorMessage": "Test Error Message"
    },
    {
      "id": "topic",
      "type": "Input.Text",
      "placeholder": "Placeholder",
      "label": "Label",
      "maxLength": 256,
      "isRequired": true,
      "errorMessage": "Test Error Message",
      "spacing": "Large"
    },
    {
      "type": "ActionSet",
      "actions": [
        {
          "type": "Action.Submit",
          "title": "Send",
          "id": "sendAction",
          "associatedInputs": "auto",
          "data": {
          "path": "/test"
          }
        }
      ]
    }
  ]
}
```

**Comment by ChetanSharma-msft** [comment]
Hello @liuguangyicao - Thanks for reporting your issue.
We will look into it and let you know the updates.

**Comment by Prasad-MSFT** [comment]
@liuguangyicao - Could you please confirm whether you are having this issue in Teams desktop client or web client and in which version of teams?  Could you please confirm which screen reader you are using whether its Narrator, JAWS, or NVDA?

**Comment by Prasad-MSFT** [comment]
We tried to repro this issue using Narrator, however the screen reader is reading the header as **App Name + Title dialog** e.g. as per the attached video Action ME pm Create Card dialog.
We tested this both in MS Teams desktop client (Microsoft Teams Version 1.6.00.364) and web client. Attaching the video for reference: 
[AccessibilityDialog.pptx](https://github.com/MicrosoftDocs/msteams-docs/files/10379940/AccessibilityDialog.pptx)
